frontend/src/index.pug

```diff
@@ -127,55 +127,61 @@ html
-                  v-bind:value="filterUntilDate", v-on:keyup.enter="updateTmpFilterUntilDate",
-                  onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
+                v-bind:value="filterUntilDate", v-on:keyup.enter="updateTmpFilterUntilDate",
+                onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
               input(v-else, type="date", v-bind:value="filterUntilDate", v-on:input="updateTmpFilterUntilDate",
-                  onkeydown="return false", v-bind:min="filterSinceDate", v-bind:max="maxDate")
+                onkeydown="return false", v-bind:min="filterSinceDate", v-bind:max="maxDate")
               label until
             .mui-textfield.summary-picker__date
               a(v-on:click="resetDateRange") Reset date range
             .summary-picker__checkboxes.summary-picker__section
               label
-                input(type="checkbox", v-model="filterBreakdown", v-on:change="setSummaryHash()").mui-checkbox.summary-picker__checkbox
+                input.mui-checkbox.summary-picker__checkbox(type="checkbox", v-model="filterBreakdown",
+                  v-on:change="setSummaryHash()")
                 span breakdown by file type
-              label.merge-group(v-bind:style="filterGroupSelection === 'groupByNone' ? { opacity:0.5 } : { opacity:1.0 }")
-                input(
-                    type="checkbox",
-                    v-model="isMergeGroup",
-                    v-bind:disabled="filterGroupSelection === 'groupByNone'"
-                ).mui-checkbox.summary-picker__checkbox
+              label.merge-group(v-bind:style="filterGroupSelection === 'groupByNone' ?\
+                { opacity:0.5 } : { opacity:1.0 }")
+                input.summary-picker__checkbox.mui-checkbox(
+                  type="checkbox",
+                  v-model="isMergeGroup",
+                  v-bind:disabled="filterGroupSelection === 'groupByNone'"
+                )
                 span merge group
         .error-message-box(v-if="Object.entries(errorMessages).length")
           .error-message-box__close-button(onclick="dismissTab(this)") &times;
           .error-message-box__message The following issues occurred when analyzing the following repositories:
           .error-message-box__failed-repo(v-for="errorBlock in errorMessages")
             i.fas.fa-exclamation.mini-font
             span.error-message-box__failed-repo--name {{ errorBlock.repoName }}
-            div.error-message-box__failed-repo--reason(v-if="errorBlock.errorMessage.startsWith('Unexpected error stack trace')")
-              span Oops, an unexpected error occurred. If you think this is due to a problem in RepoSense, please report in&nbsp;
+            .error-message-box__failed-repo--reason(
+              v-if="errorBlock.errorMessage.startsWith ('Unexpected error stack trace')"
```

Comment:
Weird space here

Reply:
before `(`

Reply:
In most of the already written code, we have broken after `(`, this is why did the same.

Reply:
Do you want me to change?

Reply:
@fzdy1914 Update on this?

Reply:
If I am correct, pug doesn't allow you to break before `(` and only after

Reply:
@Tejas2805 I think @fzdy1914 meant the spacing after your `startsWith`

Reply:
![image](https://user-images.githubusercontent.com/35621726/74452709-64316b80-4ebc-11ea-9007-5d1d6a4169bd.png)

Reply:
Yupp James made me realise that. Have updated it.